validate/setup-guides/aws/sg.md

```diff
@@ -20,7 +20,7 @@ import DocsCards from '@components/global/DocsCards';
   </style>
 </head>
 
-### Securing our EC2 Instance
+## Securing our EC2 Instance
```

Comment:
Can we remove the "like and subscribe" stuff from below? Also the note to watch the whole series. The "very important for mainnet launch" section should be updated or removed since we're after the launch so nobody really needs to know about the difference between the old network and this one.